### PR TITLE
CANVAS-93 - Enable Dependabot

### PR DESCRIPTION
**Pull request by chern**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Configuring dependabot.yml: https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```
Comment (rlorenzo) [inline .github/dependabot.yml:8]: @chern This is weekly, but what day of the week will it run? Can it be configured to be Monday?

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Configuring dependabot.yml: https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "npm"
```
Comment (rlorenzo) [inline .github/dependabot.yml:5]: @chern Does it make a difference we are using yarn? Will it create pull requests that update our yarn.lock file?

File: .github/dependabot.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    directory: "/"
+    schedule:
+      interval: "weekly"
+
+  - package-ecosystem: "github-actions"
+    directory: "/"
```
Comment (rlorenzo) [inline .github/dependabot.yml:11]: @chern Is this really the root directory? Not .github/workflows?

File: .github/dependabot.yml (hunk 1 above)
Comment (chern) [reply .github/dependabot.yml:8]: By default, setting it to weekly sets it to run on Monday: https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates#scheduleinterval

File: .github/dependabot.yml (hunk 3 above)
Comment (chern) [reply .github/dependabot.yml:11]: @rlorenzo Dependabot will check inside .github/workflows when the directory is set as "/" (https://docs.github.com/en/github/administering-a-repository/keeping-your-actions-up-to-date-with-github-dependabot#enabling-github-dependabot-version-updates-for-actions)

File: .github/dependabot.yml (hunk 2 above)
Comment (chern) [reply .github/dependabot.yml:5]: @rlorenzo Yarn isn't listed explicitly as a package-ecosystem (https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates#package-ecosystem), but Dependabot's own website says it supports Yarn lockfiles: https://dependabot.com/javascript/

**Pull request merged**